--- sharelatex_ori/navbar-marketing.pug	2023-11-07 13:36:10.000000000 +0100
+++ sharelatex/navbar-marketing.pug	2024-04-10 19:20:01.895583224 +0200
@@ -52,8 +52,6 @@
 										a(href="/admin") Manage Site
 									li
 										a(href="/admin/user") Manage Users
-									li
-										a(href="/admin/project") Project URL Lookup
 								if canDisplayAdminRedirect
 									li
 										a(href=settings.adminUrl) Switch to Admin
