--- sharelatex_ori_5/navbar-website-redesign.pug	2024-06-10 10:22:04
+++ sharelatex_ori_5/navbar-website-redesign.fixed.pug	2024-06-10 10:27:10
@@ -1,14 +1,13 @@
 nav.navbar.navbar-default.navbar-main.website-redesign-navbar
 	.container-fluid
 		.navbar-header
-			if (typeof(suppressNavbarRight) == "undefined")
-				button.navbar-toggle.collapsed(
-					type="button",
-					data-toggle="collapse",
-					data-target="#navbar-main-collapse"
-					aria-label="Toggle " + translate('navigation')
-				)
-					i.fa.fa-bars(aria-hidden="true")
+			button.navbar-toggle.collapsed(
+				type="button",
+				data-toggle="collapse",
+				data-target="#navbar-main-collapse"
+				aria-label="Toggle " + translate('navigation')
+			)
+				i.fa.fa-bars(aria-hidden="true")
 			- var enableUpgradeButton = projectDashboardReact && usersBestSubscription && usersBestSubscription.type === 'free'
 			if (enableUpgradeButton)
 				a.btn.btn-primary.pull-right.me-2.visible-xs(
@@ -26,43 +25,14 @@
 			else
 				a(href='/', aria-label=settings.appName).navbar-brand
 
-		- var canDisplayAdminMenu = hasAdminAccess()
-		- var canDisplayAdminRedirect = canRedirectToAdminDomain()
-		- var canDisplaySplitTestMenu = hasFeature('saas') && (canDisplayAdminMenu || (getSessionUser() && getSessionUser().staffAccess && (getSessionUser().staffAccess.splitTestMetrics || getSessionUser().staffAccess.splitTestManagement)))
-		- var canDisplaySurveyMenu = hasFeature('saas') && canDisplayAdminMenu
-
 		if (typeof(suppressNavbarRight) == "undefined")
 			.navbar-collapse.collapse#navbar-main-collapse
 				ul.nav.navbar-nav.navbar-right
-					if (canDisplayAdminMenu || canDisplayAdminRedirect || canDisplaySplitTestMenu)
-						li.dropdown.subdued
-							a.dropdown-toggle(
-								href="#",
-								role="button",
-								aria-haspopup="true",
-								aria-expanded="false",
-								data-toggle="dropdown"
-							)
-								| Admin
-								span.caret
-							ul.dropdown-menu
-								if canDisplayAdminMenu
-									li
-										a(href="/admin") Manage Site
-									li
-										a(href="/admin/user") Manage Users
-									li
-										a(href="/admin/project") Project URL Lookup
-								if canDisplayAdminRedirect
-									li
-										a(href=settings.adminUrl) Switch to Admin
-								if canDisplaySplitTestMenu
-									li
-										a(href="/admin/split-test") Manage Feature Flags
-								if canDisplaySurveyMenu
-									li
-										a(href="/admin/survey") Manage Surveys
+					if (getSessionUser() && getSessionUser().isAdmin)
+						li
+							a(href="/admin") Admin
 
+
 					// loop over header_extras
 					each item in nav.header_extras
 						-
